### DISCUSSION TITLE
Why in v12.0.0 libx265 is not present in pyav available codecs?

### DISCUSSION THREAD

**Discussion by alfredraphaelb** [PyAV-Org/PyAV#1370] [1. Help] [answered]
Why in v12.0.0 libx265 is not present in pyav available codecs?

**Comment by WyattBlue** [comment, marked as answer]
We include libx265 in the PyAV binary wheels. You or your distro might have used a version of PyAV that doesn't include GPLv3 deps. Install PyAV with `pip install av`
